### DISCUSSION TITLE
Quarkus Meetup Munich

### DISCUSSION THREAD

**Discussion by d135-1r43** [quarkusio/quarkus#30927] [Events]
At exentra, we have taken note of the significant attendance at the Spring Boot Meetup in Munich and have considered the possibility of establishing a Quarkus Meetup in the Munich metropolitan area. We are curious to know if there are any developers based in Munich who would be interested in participating in this endeavor? I know from past meetups that Quarkus is quite strong at BMW?

**Comment by heubeck** [comment]
Servus @d135-1r43,
what do you have in mind?

How about connecting the Java User Groups around (Munich, Regensburg, Augsburg, Oberland, Nuremberg, Ingolstadt) for Quarkus focused meetups?

Maybe @wpernath can act as local insighter?

**Comment by d135-1r43** [comment]
A potential idea is to host a meetup on a quarterly basis, featuring two talks on Quarkus and its related technologies, opportunities for networking, and pizza. The venue should be located in either Munich or Paffenhofen (for the convenience of those from Ingolstadt and Augsburg). I will make efforts to engage and recruit more interested developers through JUG M, LJUG M, and JUG IN.

**Reply by heubeck** [reply]
sounds great, meet in the gravity center of the (confirmed) participants for pizza is always a good idea ;)

**Reply by sbeugen** [reply]
Hi. I am from Comsysto Reply in Munich and we are the organizers of the Lightweight Java User Group Munich. I will talk to my colleagues and see, how we can support you.

**Comment by andrejpetras** [comment]
I would be interested in the Munich group. We do have strong groups of Quarkus developers in Telekom projects. I will also ask colleagues who are working on BMW projects.

**Comment by preslavrachev** [comment]
A Quarkus meetup in Munich would be fantastic! I'm all for it 👍

**Comment by jfey60** [comment]
I did introduce Quarkus in quite a few industry projects during the last 2..3 years. Nobody knew about it before (usually Spring Boot folks), but everyone seems to be happy with it after all. A Quarkus Meetup in Munich would be nice. I am using a lot of Golang based backend stuff these days, but the next Quarkus-based activity for a startup is imminent. I was using Grails a lot years ago.

**Comment by mariasalcedo** [comment]
Hi! What do we need to do in order to make this possible? Also, bringing the "Quarkus World Tour" to any of the JUGs in and around Munich could also be a possibility? I am all for it, and at your disposal to participate in this event organization :)

**Comment by ghost** [comment]
Hey everyone! I had the same idea, that Quarkus deserves its own exclusive User Group, and now it's a reality! Introducing the Quarkus User Group Munich! We've just scheduled our first event. Check out our website: www.qugm.de. I've been in touch with Red Hat, and we've got some incredible speakers, ideas and topics lined up for upcoming events. Any ideas or concerns are always welcome!

**Comment by stefanschaller** [comment]
Any news, upcoming events? Looks like the Group is dead 😕  @vinzent-ferreira  @d135-1r43

**Comment by stefanschaller** [comment]
We could arrange a location for a meetup in Munich including drinks next to the Hauptbahnhof. 
How about that?

**Reply by d135-1r43** [reply]
@vinzent-ferreira is the owner of the Meetup group. I would be happy to participate, either as a speaker, a supporter or simply buy coming with my collegues. 

Should we start a Signal group for further planning?
